Resources/public/css/layout.css

```diff
@@ -355,6 +355,10 @@ body.fixed .content-header .navbar.stuck {
     border-radius: 0;
 }
 
+.box .nav-stacked > li {
```

Comment:
The original idea behind ``layout.css`` and ``styles.css`` was (5 years ago ;) to allow overwrite colors without breaking the structure defined in layout.

@Soullivaneuh maybe we should fix for the search view only by adding a specific class